config/initializers/rack_attack.rb

```diff
@@ -32,21 +32,23 @@ def remote_ip
     request.path == "/check"
   end
 
-  throttle("Login attempts by ip", limit: 5, period: 20.seconds) do |request|
-    if request.path == "/users/sign_in" && request.post?
-      request.remote_ip
+  unless Rails.env.performance?
```

Comment:
Maybe we can try setting `Rack::Attack.enabled = false` in the environment config instead, to avoid wrapping the settings here?

Reply:
used that instead as I did not know about it